docs/plugins/parentwork.rst

```diff
@@ -18,7 +18,13 @@ example, all the movements of a symphony. This plugin aims to solve this
 problem by also fetching the parent work, which would be the whole symphony in
 this example.
 
-This plugin adds five tags:
+It uses the ``mb_workid`` tag to retrieve the *parent work* and stores the
+``mb_workid`` as ``parentwork_workid_current``. When one executes 
+``beet parentwork`` it checks if ``mb_workid`` has changed by comparing it
+with ``parentwork_workid_current``, if it has changed or if ``force`` is
+enabled it fetches the *parent work* again.
```

Comment:
Here's a perhaps more direct way to tell the user why this works this way:

> The plugin can detect changes in `mb_workid` so it knows when to re-fetch other metadata, such as `parentwork`. To do this, when it first gets the parent work, it stores it *both* in `mb_parentworkid` *and* in a special bookkeeping field `parentwork_workid_current`. Then, you can later change `mb_parentworkid` (with the `modify` command, for example) and re-run the plugin. It will notice that the two tags no longer match and will re-fetch all the parent work metadata. You can also use the `force` option to re-fetch metadata for _all_ music.

Reply:
Actually, it's not how it works. ```parentwork_workid_current``` is a copy of ```mb_workid```, not a copy of ```mb_parentworkid```. I'm adapting, you tell me if it is fine.

Reply:
Great; yes, that sounds good!